developer/src/kmc-convert/src/keylayout-to-kmn/keylayout-file-reader.ts

```diff
@@ -11,53 +11,123 @@ import { CompilerCallbacks } from "@keymanapp/developer-utils";
 import { XMLParser } from 'fast-xml-parser';
 import { util } from '@keymanapp/common-types';
 import { ConverterMessages } from '../converter-messages.js';
+import { SchemaValidators } from '@keymanapp/common-types';
+import { KeylayoutXMLSourceFile } from '../../../common/web/utils/src/types/keylayout/keylayout-xml.js';
```

Comment:
```suggestion
import { KeylayoutXMLSourceFile } from '@keymanapp/developer-utils';
```

otherwise you're importing across package boundaries here